include/trackcpp/passmethods.hpp

```diff
@@ -259,11 +281,21 @@ Status::type pm_str_mpole_symplectic4_pass(Pos<T> &pos, const Element &elem,
   double k2 = sl * KICK2;
   const std::vector<double> &polynom_a = elem.polynom_a;
   const std::vector<double> &polynom_b = elem.polynom_b;
+  double d_factor = 0; // quantum excitation scale factor
+
+  if (accelerator.quantdiff_on){
```

Comment:
I think that if `radiation_on` is off we don't need to calculate these constants, right?:

```c++
if (accelerator.radiation_on & accelerator.quantdiff_on){
```

Reply:
Fixed, thanks!